.eslintrc.json

```diff
@@ -0,0 +1,59 @@
+{
+  "plugins": ["jest", "@typescript-eslint"],
```

Comment:
I'm not using jest

Reply:
will remove when we get to that bit - no tests were added either, I was just pulling in bits of repo infrastructure from the github actions one